posthog/async_migrations/migrations/0006_persons_and_groups_on_events_backfill.py

```diff
@@ -1,474 +1,23 @@
-                    REPLACE PARTITION tuple() FROM {settings.CLICKHOUSE_DATABASE}.groups
-                """,
-                rollback=None,
-                per_shard=True,
-            ),
-            AsyncMigrationOperation(
-                fn=lambda query_id: run_optimize_table(
-                    unique_name="0006_persons_and_groups_on_events_backfill_person",
-                    query_id=query_id,
-                    table_name=f"{settings.CLICKHOUSE_DATABASE}.{TEMPORARY_PERSONS_TABLE_NAME}",
-                    final=True,
-                    deduplicate=True,
-                    per_shard=True,
-                )
-            ),
-            AsyncMigrationOperation(
-                fn=lambda query_id: run_optimize_table(
-                    unique_name="0006_persons_and_groups_on_events_backfill_pdi2",
-                    query_id=query_id,
-                    table_name=f"{settings.CLICKHOUSE_DATABASE}.{TEMPORARY_PDI2_TABLE_NAME}",
-                    final=True,
-                    deduplicate=True,
-                    per_shard=True,
-                )
-            ),
-            AsyncMigrationOperation(
-                fn=lambda query_id: run_optimize_table(
-                    unique_name="0006_persons_and_groups_on_events_backfill_groups",
-                    query_id=query_id,
-                    table_name=f"{settings.CLICKHOUSE_DATABASE}.{TEMPORARY_GROUPS_TABLE_NAME}",
-                    final=True,
-                    deduplicate=True,
-                    per_shard=True,
-                )
-            ),
-            AsyncMigrationOperationSQL(
-                sql=f"""
-                    ALTER TABLE {settings.CLICKHOUSE_DATABASE}.{TEMPORARY_PDI2_TABLE_NAME} {{on_cluster_clause}}
-                    DELETE WHERE is_deleted = 1 OR person_id IN (
-                        SELECT id FROM {TEMPORARY_PERSONS_TABLE_NAME} WHERE is_deleted=1
-                    )
-                """,
-                sql_settings={"mutations_sync": 2},
-                rollback=None,
-                per_shard=True,
-            ),
-            AsyncMigrationOperationSQL(
-                sql=f"""
-                    ALTER TABLE {settings.CLICKHOUSE_DATABASE}.{TEMPORARY_PERSONS_TABLE_NAME} {{on_cluster_clause}}
-                    DELETE WHERE is_deleted = 1
-                """,
-                sql_settings={"mutations_sync": 2},
-                rollback=None,
-                per_shard=True,
-            ),
-            AsyncMigrationOperation(fn=self._create_dictionaries, rollback_fn=self._clear_temporary_tables),
-            AsyncMigrationOperation(fn=self._run_backfill_mutation),
-            AsyncMigrationOperation(fn=self._wait_for_mutation_done),
-            AsyncMigrationOperation(fn=lambda query_id: self._postcheck(query_id)),
-            AsyncMigrationOperation(fn=self._clear_temporary_tables),
-        ]
+    posthog_max_version = "1.39.1"
```

Comment:
keep this consistent with 0007 or drop

Reply:
afaik we don't check the max_version on auto-completion, but it can throw people off in the future migrations tab